qa/test-harness/test-harness.py

```diff
@@ -8,7 +8,8 @@
 from argparse import ArgumentParser
 from kafka import KafkaConsumer
 from pathlib import Path
-from odevalidator import TestCase
+from odevalidator.validator import TestCase
```

Comment:
Should be `from odevalidator import TestCase`

Reply:
Relative import does not work for running odevalidator.validator tester. For now, non-relative imports seem to work everywhere. Will revisit relative imports at a later time.